src/ansys/mapdl/core/misc.py

```diff
@@ -425,3 +429,425 @@ def check_valid_start_instance(start_instance):
         )
 
     return start_instance.lower() == "true"
+
+
+def requires_update(requires_update=False):
```

Comment:
Docstring... I know it's just for developers, but developers 5 years down the line shouldn't have to look up how this function is used to use it.

Reply:
You are totally right.